crates/bench/benches/uv.rs

```diff
@@ -47,9 +47,10 @@ fn resolve_warm_airflow(c: &mut Criterion<WallTime>) {
     let venv = PythonEnvironment::from_root("../../.venv", cache).unwrap();
     let client = &RegistryClientBuilder::new(cache.clone()).build();
     let manifest = &Manifest::simple(vec![
-        Requirement::from(pep508_rs::Requirement::from_str("apache-airflow[all]").unwrap()),
+        Requirement::from(pep508_rs::Requirement::from_str("apache-airflow[all]==2.9.2").unwrap()),
         Requirement::from(
-            pep508_rs::Requirement::from_str("apache-airflow-providers-apache-beam>3.0.0").unwrap(),
+            pep508_rs::Requirement::from_str("apache-airflow-providers-apache-beam==5.7.1")
+                .unwrap(),
```

Comment:
I'd leave that constraint as it is, the difficulty for that resolution is partially unifying those two constraints; with the exclude newer it will still be stable